sentry-sidekiq/lib/sentry/sidekiq/sentry_context_middleware.rb

```diff
@@ -26,8 +26,18 @@ def call(_worker, job, queue)
         scope.set_span(transaction) if transaction
 
         begin
-          yield
-        rescue
+          Sentry.with_child_span(op: "queue.process", description: "Process #{worker.class.name}") do |span|
+            # Set span data
+            if span
+              span.set_data("messaging.message.id", job["jid"])
+              span.set_data("messaging.destination.name", queue)
+              span.set_data("messaging.message.receive.latency", ((Time.now.to_f - job["enqueued_at"]) * 1000).to_i)
+              span.set_data("messaging.message.retry.count", job["retry_count"] || 0)
+            end
+
+            yield
+          end
+        rescue => ex
```

Comment:
Not sure it matters, but you don't use this exception object, so you may want to not even bother with creating it?

Reply:
You're right - Fixed.